posthog/models/person.py

```diff
@@ -76,6 +76,8 @@ def merge_people(self, people_to_merge: List["Person"]):
 class PersonDistinctId(models.Model):
     class Meta:
         constraints = [models.UniqueConstraint(fields=["team", "distinct_id"], name="unique distinct_id for team")]
+        # Also has an index on properties -> email, built concurrently
```

Comment:
We can track this on the model.

See https://github.com/PostHog/posthog/commit/e03477b7d45e6d94dec2ebddd31f9c16910ea257 - with that it should handle it fine.

Reply:
Annoyingly doesn't work given it's a JSONField index.